### PR TITLE
fix relative imports

### LINKED ISSUE #187: convert relative imports to absolute
When I wrote the first version of this package, I added relative imports. then, I realized their usage is [discouraged](https://peps.python.org/pep-0008/#imports). since we still have a lot of them, it's unclear for developers what's the rule here, so we should migrate them so there's no ambiguity

### PR DESCRIPTION
**Pull request by WSShawn**
## Describe your changes
Update the relative imports in sklearn-evaluation repo to absolute imports.

## Issue ticket number and link
Closes #187 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

----
:books: Documentation preview :books:: https://sklearn-evaluation--228.org.readthedocs.build/en/228/

### CODE REVIEW

**Comment by idomic** [comment]
@WSShawn Closes #x

Add the correct issue number it closes for the PR.

**Review by idomic — CHANGES_REQUESTED** [review]
Looks good, please go over the docs and make sure there's no funky stuff going on there. You have a link inside this PR.

**Comment by WSShawn** [comment]
check finished! nothing weird in the page.

**Comment by yafimvo** [comment]
@WSShawn Please refer to the original issue (closes #X)

**Comment by WSShawn** [comment]
Sorry, I'm not sure what it means to close #X and also what is the original issue. Thank you!

**Comment by WSShawn** [comment]
Closes #187

**Comment by WSShawn** [comment]
Thanks for the reminder!

**Review by yafimvo — CHANGES_REQUESTED** [review]
@edublancas @idomic @WSShawn 
We changed the relative imports to absolute, but do we want to change these as well (the imports in `__init__.py`)?

it makes the code a bit longer and hard to maintain (e.g lint issue or if we will rename the `plot` directory).

in the [PEP8 guidelines ](https://peps.python.org/pep-0008/#imports) they write:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

> ``` python
> import mypkg.sibling
> from mypkg import sibling
> from mypkg.sibling import example
> ```
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:
> 
> ``` python
> from . import sibling
> from .sibling import example
> ```
> Standard library code should avoid complex package layouts and always use absolute imports.
>

File: src/sklearn_evaluation/plot/__init__.py [hunk 1]
```diff
@@ -2,32 +2,35 @@
...
     silhouette_analysis_from_results,
 )
-from .regression import residuals, prediction_error, cooks_distance
-from .target_analysis import target_analysis
-from .calibration import calibration_curve, scores_distribution
-from .classification_report import classification_report, ClassificationReport
-from .ks_statistics import ks_statistic
-from .cumulative_gain_lift_curve import cumulative_gain, lift_curve
-from .feature_ranking import Rank1D, Rank2D
+from sklearn_evaluation.plot.regression \
```
Comment (yafimvo) [inline src/sklearn_evaluation/plot/__init__.py:24]: @edublancas @idomic @WSShawn 
We changed the relative imports to absolute, but do we want to change these as well (the imports in `__init__.py`)?

it makes the code a bit longer and hard to maintain (e.g lint issue or if we will rename the `plot` directory).

in the [PEP8 guidelines ](https://peps.python.org/pep-0008/#imports) they write:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

> ``` python
> import mypkg.sibling
> from mypkg import sibling
> from mypkg.sibling import example
> ```
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:
> 
> ``` python
> from . import sibling
> from .sibling import example
> ```
> Standard library code should avoid complex package layouts and always use absolute imports.
>

**Review by yafimvo — CHANGES_REQUESTED** [review]

File: src/sklearn_evaluation/plot/__init__.py (hunk 1 above)
Comment (WSShawn) [reply src/sklearn_evaluation/plot/__init__.py:24]: So we want to change the absolute back to explicit relative in _init_.py?

Comment (yafimvo) [reply src/sklearn_evaluation/plot/__init__.py:24]: Let's see what @edublancas and @idomic think

**Comment by neelasha23** [comment]
lgtm! Docs also look good overall.
Also, the `Closes #x` needs to be mentioned in the `Issue ticket number and link` section of the PR.
Please add some one-liner or relevant descriptions in the `Describe your changes` also when opening a PR.

File: src/sklearn_evaluation/plot/__init__.py (hunk 1 above)
Comment (edublancas) [reply src/sklearn_evaluation/plot/__init__.py:24]: let's keep them as absolute, we should be consistent (all absolute or all relative)

**Review by yafimvo — APPROVED** [review]

**Review by edublancas — APPROVED** [review]

**Pull request merged**